source/dagger/gameplay/racing/racing_game_logic.cpp

```diff
@@ -39,7 +57,19 @@ void RacingCollisionsLogicSystem::Run()
 
             if (col.colided)
             {
-                m_Restart = true;
+                col.colided=false;
+                if(player.pause>0) player.pause--;
+                else
+                {    
+                     
+                    player.pause=200;
+                    player.strike++;
+                    racing_game::CreateStrike(20.f,player.strike);
+                    
+                    if(player.strike==3) m_Restart = true;       
```

Comment:
it's better to have habit to use {} everywhere. even in cases like that. 
it makes code more readable